rules/crowdstrike_rules/crowdstrike_real_time_response_session.py

```diff
@@ -0,0 +1,20 @@
+from panther_base_helpers import deep_get
+
+
+def rule(event):
+    return deep_get(event, "unknown_payload", "ExternalApiType") == "Event_RemoteResponseSessionStartEvent"
+
+
+def title(event):
+    user_name = deep_get(event, "unknown_payload", "UserName")
```

Comment:
not required here, but recommend adding defaults to deep_get here as well

Reply:
Added default option to deep_get().